### PR TITLE
Update AboutUs and README

### LINKED ISSUE #14: Update project website: AboutUs, ContactUs, and README
- [X] AboutUs
- [X] ContactUs
- [X] README

### PR DESCRIPTION
**Pull request by dinhnhobao**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -15,10 +15,29 @@ ifndef::env-github[]
...
+* Save important content from a webpage for *offline* use, and *annotate* it right away.
+* Set *reminders* for websites. Never forget to pay those pesky e-bills again!
+* Sort and tag your bookmarks automatically with our intelligent *auto-tagging* system. Keeping your bookmarks organized is easy.
+////
 * It can get your management of tasks done faster than traditional GUI apps, if you can type fast.
 
+****
+
+// Target user
+Target User: please fill in...
```
Comment (alushingg) [inline README.adoc]: I think it can be NUS students

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -284,7 +284,7 @@ Address book data are saved in the hard disk automatically after any command tha
 There is no need to save manually.
 
 === Upcoming `[coming in v2.0]`
-These features will be available in the next version of Mark.
+These features will be available in the next version of Mod Manager.
```
Comment (alushingg) [inline docs/UserGuide.adoc:287]: For line 26 of this document, i think change Mark to Mod Manager. From line 33, 51 to 54, can help change ModManager to Mod Manager. Thanks.

Comment (dinhnhobao) [reply docs/UserGuide.adoc:287]: Resolved. Thanks for the feedback!

File: README.adoc (hunk 1 above)
Comment (dinhnhobao) [reply README.adoc]: Okay

**Pull request merged**